cirq-core/cirq/contrib/quimb/Contract-a-Grid-Circuit.ipynb

```diff
@@ -12,7 +12,7 @@
     "    import cirq\n",
     "except ImportError:\n",
     "    print(\"installing cirq...\")\n",
-    "    !pip install --quiet cirq --pre\n",
+    "    !pip install --quiet cirq\n",
```

Comment:
Might be nice to combine this with the following code which tries to import quimb and then installs more packages:
```python
try:
    import cirq
    import quimb
except ImportError:
    print("installing cirq-core[contrib]...")
    !pip install --quiet cirq-core[contrib]
    print("installed cirq-core[contrib].")
```